apps/src/gamelab/GameLab.js

```diff
@@ -286,6 +323,76 @@ GameLab.prototype.afterInject_ = function (config) {
 };
 
 /**
+ * Return whether to render a footer inside the game space
+ * (must sync with game.rb's owns_footer_for_share?)
+ * (Currently, this is disabled until we understand how we want to render)
+ */
+GameLab.prototype.shouldRenderFooter_ = function () {
+  return false;
+  // return this.studioApp_.share;
+};
+
+/**
+ * Render footer inside the game space like applab does (currently disabled)
+ */
+GameLab.prototype.renderFooterInSharedGame_ = function () {
+  var divGameLab = document.getElementById('divGameLab');
+  var footerDiv = document.createElement('div');
+  footerDiv.setAttribute('id', 'footerDiv');
+  divGameLab.parentNode.insertBefore(footerDiv, divGameLab.nextSibling);
+
+  var menuItems = [
+    {
+      text: commonMsg.reportAbuse(),
+      link: '/report_abuse',
+      newWindow: true
+    },
+    {
+      text: msg.makeMyOwnGame(),
+      link: '/projects/gamelab/new',
+      hideOnMobile: true
+    },
+    {
+      text: commonMsg.openWorkspace(),
+      link: location.href + '/view'
+    },
+    {
+      text: commonMsg.copyright(),
+      link: '#',
+      copyright: true
+    },
+    {
+      text: commonMsg.privacyPolicy(),
+      link: 'https://code.org/privacy',
+      newWindow: true
+    }
+  ];
+  if (dom.isMobile()) {
+    menuItems = menuItems.filter(function (item) {
+      return !item.hideOnMobile;
+    });
+  }
+
+  ReactDOM.render(React.createElement(window.dashboard.SmallFooter,{
+    i18nDropdown: '',
+    copyrightInBase: false,
+    copyrightStrings: this.copyrightStrings,
+    baseMoreMenuString: commonMsg.builtOnCodeStudio(),
+    rowHeight: FOOTER_HEIGHT,
+    style: {
+      fontSize: 18
+    },
+    baseStyle: {
+      width: $("#divGameLab").width(),
+      paddingLeft: 0
+    },
+    className: 'dark',
+    menuItems: menuItems,
+    phoneFooter: true
+  }), footerDiv);
+};
```

Comment:
I think it is possible, but it does differ slightly in one menu item. Also, I don't think I'm going to end up using this in gamelab because we need the footer to appear below the d-pad.

So, that's why this is disabled for now.